addons/mail/static/src/owl/components/activity_box/activity_box.scss

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_ActivityBox_title {
+    display: flex;
+    align-items: center;
+    flex: 0 0 auto;
+    margin-top: 15px;
+    margin-bottom: 20px;
+}
+
+.o_ActivityBox_titleBadge {
+    padding: 0 5px;
+}
+
+.o_ActivityBox_titleLine {
+    flex: 1 1 auto;
+    width: auto;
+}
+
+.o_ActivityBox_titleText {
+    padding: 0 10px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_ActivityBox_title {
+    font-weight: bold;
+}
+
+.o_ActivityBox_titleBadge {
+    font-size: 11px;
```

Comment:
can we work with one of the `$font-size` -base, -lg or -sm ?

Reply:
base it too big and sm too small imho... (from left to right : sm, 11px, base, lg)

![image](https://user-images.githubusercontent.com/48204996/75667386-e15c2f00-5c77-11ea-886f-9d4ff6c4bfd1.png)